app/views/content/a-day-in-the-life-of-a-teacher.md

```diff
@@ -22,22 +22,22 @@
       text: "I like to get in early so I can get organised — in teaching, preparation is everything."
       name: "Sarah"
       job_title: "Primary school teacher"
```

Comment:
@garethwilliamsdesign @martyn-w Can we merge the name / job title onto one line as we do on the SEND page?

Otherwise this page looks much better now!